src/utils.rs

```diff
@@ -76,3 +77,34 @@ pub fn find_workspace(manifest: &Path, name: &str) -> Result<Option<PathBuf>, Er
     }
     Ok(None)
 }
+
+/**
+ * Search for .cargo/config.toml file
+ *
+ * - Receive the workspace root path
+ */
+pub fn find_cargo_config(path: &Path) -> Result<Option<PathBuf>, Error> {
+    let path = dunce::canonicalize(path)?;
+    for config_path in path
+        .ancestors()
+        .map(|dir| dir.join(".cargo/config.toml"))
+        .filter(|dir| dir.is_file())
+    {
+        return Ok(Some(config_path));
+    }
+    Ok(None)
```

Comment:
This is returning a single file now, no need for a loop. Can be simplified with `find`:

```suggestion
    path
        .ancestors()
        .map(|dir| dir.join(".cargo/config.toml"))
        .find(|dir| dir.is_file())
```